crm_win_reason/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
```

Comment:
```suggestion
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
```
https everywhere :)

anyway there's no need for headers in `__init__.py` files